InputTime/index.jsx

```diff
@@ -84,8 +86,8 @@ const InputTime = ({
       marginBottom: 0,
       marginLeft: 0,
       padding: 0,
-      '-webkit-appearance': 'none',
-      '-moz-appearance': 'none',
+      WebkitAppearance: 'none',
```

Comment:
I see what you're saying here, the default styling makes the select element wider :smile: 

As far as the `fontFamily` goes -- setting that explicitly on the `Text` component would likely fix that -- would be good to do that in a followup commit on buffer components :smile_cat: